### PR TITLE
Add teeracle pallet 

### PR DESCRIPTION
**Pull request by echevrier**
used by the exchange rate oracle. integritee-network/worker#292

### CODE REVIEW

File: runtime/Cargo.toml [hunk 1]
```diff
@@ -114,5 +118,6 @@ runtime-benchmarks = [
 	"pallet-treasury/runtime-benchmarks",
 	"pallet-vesting/runtime-benchmarks",
 	"pallet-utility/runtime-benchmarks",
+	#"pallet-teeracle/runtime-benchmarks",
```
Comment (haerdib) [inline runtime/Cargo.toml]: Should this pallet not be benchmarked?

File: runtime/Cargo.toml [hunk 2]
```diff
@@ -10,6 +10,8 @@ version = '0.9.3'
 [package.metadata.docs.rs]
 targets = ["x86_64-unknown-linux-gnu"]
 
+# local dependencies
```
Comment (haerdib) [inline runtime/Cargo.toml]: Is this comment still valid?

**Review by haerdib — COMMENTED** [review]
Does this pallet not need any benchmark weights?

File: runtime/Cargo.toml (hunk 1 above)
Comment (echevrier) [reply runtime/Cargo.toml]: Yes. I will comment it out in the next task. https://github.com/integritee-network/integritee-node/issues/65

File: runtime/Cargo.toml (hunk 2 above)
Comment (echevrier) [reply runtime/Cargo.toml]: Not really. I will remove it

**Review by haerdib — APPROVED** [review]
LGTM 👍

**Comment by brenzi** [comment]
love the freestyle wordplay in the naming 👍

**Pull request merged**